core/arch/arm/include/mm/core_mmu.h

```diff
@@ -116,12 +116,10 @@ struct core_mmu_phys_mem {
 		__used __section("phys_mem_map_section") = \
 		{ #addr, (type), (addr), (size) }
 
-#define register_sdp_mem(pa, sz) \
-	static const struct core_mmu_phys_mem __phys_sdp_mem_ ## pa\
-		__used __section("phys_sdp_mem_section") = { \
-			.name = #pa, .addr = (pa), .size = (sz), \
-			.type = MEM_AREA_SDP_MEM, \
-	}
+#define register_sdp_mem(label, pa, sz) \
+	static const struct core_mmu_phys_mem __phys_sdp_mem_ ## label\
```

Comment:
yes, right!
however i failed to get __LINE__ being interpreted as a value by `##` magic operator.